frontend/src/Frontend/ReplGhcjs.hs

```diff
@@ -142,7 +150,7 @@ walletSidebar sidebarExtra = elAttr "div" ("class" =: "sidebar") $ do
 -- | Get the routes to the icon assets for each route
 routeIcon :: R FrontendRoute -> (Text, Text)
 routeIcon = \case
-  FrontendRoute_Main :/ () -> (static @"img/menu/contracts.png", static @"img/menu/contracts_highlighted.png")
+  FrontendRoute_Contracts :/ Nothing -> (static @"img/menu/contracts.png", static @"img/menu/contracts_highlighted.png")
```

Comment:
Doesn't this yield a warning? We're not handling `FrontendRoute_Contracts :/ Just r`

Reply:
There's a wildcard further down, but I just stuck wildcards after `:/`, seems to make more sense